pkg/otlp/model/attributes/gcp/gcp.go

```diff
@@ -39,13 +68,10 @@ func HostnameFromAttributes(attrs pcommon.Map) (string, bool) {
 
 // HostInfoFromAttributes gets GCP host info from attributes following
 // OpenTelemetry semantic conventions
-func HostInfoFromAttributes(attrs pcommon.Map) (hostInfo *HostInfo) {
+func HostInfoFromAttributes(attrs pcommon.Map, usePreviewRules bool) (hostInfo *HostInfo) {
```

Comment:
Can we clarify in the docs what this second argument means? Both here and upstairs?

Reply:
👍 It's related to open-telemetry/opentelemetry-collector-contrib#10424, I will add a few details about it

Reply:
I added some details here https://github.com/DataDog/datadog-agent/blob/96c14d69396635ac2b6533e686ad0f8a2de7b045/pkg/trace/config/config.go#L63-L66 does that make sense? This will be a temporary argument

Reply:
Ok, it's fine. It's not totally clear to me what it means, but I don't want to torture you with explanations if this is just a temporary change :) Resolving.